src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
+    private int valueBuy = 0;
+    private int valueSupply = 0;
+    private int result = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String fromFile = fromFileName;
+        String toFile = toFileName;
+
+        readingData(fromFile);
+        printToFile(valueSupply, valueBuy, result, toFile);
+    }
+
+    public void readingData(String fromFileName) {
```

Comment:
```suggestion
    public int[] readDataFromFile(String fromFileName) {
```
краще замість void повернути результат виконання якщо метод може його згенерити, так код буде простіше тестувати в майбутеньому і перевикористовувати. 
Тоді можна буде з рівня класу викосити змінні valueSupply, valueBuy, result і це допоможе зробити код більш гнучким